iati/tests/test_resources.py

```diff
@@ -50,193 +177,492 @@ class TestResourceFolders(object):
         (iati.Version('1.01'), '1-01'),
         ('1', '1'),
         ('2', '2'),
-        (iati.version.STANDARD_VERSION_ANY, 'version-independent')
+        (iati.version.STANDARD_VERSION_ANY, iati.resources.PATH_VERSION_INDEPENDENT)
     ])
     @pytest.mark.latest_version('2.02')
-    def test_folder_name_for_version(self, version, expected_version_foldername):
-        """Check that expected components are present within folder paths."""
-        path = iati.resources.folder_name_for_version(version)
+    def test_version_folder_name_generation_known(self, version, expected_version_foldername):
+        """Check that the correct folder name is returned for known version numbers."""
+        folder_name = iati.resources.folder_name_for_version(version)
 
-        assert expected_version_foldername == path
+        assert expected_version_foldername == folder_name
 
-    @pytest.mark.parametrize('version', [
-        '1.00',
-        1,
-        1.01,  # A version must be specified as a string
-        'string'
-    ])
-    def test_folder_name_for_version_invalid_version(self, version):
-        """Check that an invalid version of the Standard raises a ValueError exception."""
+    def test_version_folder_name_generation_unknown(self, std_ver_all_mixedinst_valid_unknown):
```

Comment:
The naming structure of test functions in this class isn't consistent (the position of `version` in the name varies).

Reply:
7ec75b154ac36bb93db056e4e2dc75674456cda1